src/CORS/samples/SampleDestination/Program.cs

```diff
@@ -11,12 +12,30 @@ public class Program
     {
         public static void Main(string[] args)
         {
+            Type startupType = null;
+
+            var startup = Environment.GetEnvironmentVariable("CORS_STARTUP");
+            switch (startup)
+            {
+                case "Startup":
```

Comment:
Could we default to this? It's useful to run this app outside of a test when iterating on the code and being unable to `dotnet run` the app might become quickly annoying